### ISSUE TITLE
Don't change font by default

### ISSUE THREAD

**Issue by jacksonludwig** [rougier/nano-theme#4]
The theme should just change the color theme by default, in my opinion. Changing font size especially doesn't make much sense as the font size is going to be either too big or too small on most people's screens.

What would you think of removing font changes? Or perhaps it would make sense to include them in your `nano-setup` function?

**Comment by rougier** [comment]
You're right. The plan is to have an option to disable font family/size. Just need to do it!
